backend/prisma/migrations/20211221182939_add_score_entry/migration.sql

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
meiner Meinung nach kannst du alle migrations in ein `init` packen. wir sind ja noch lange nicht "produktiv" und lokal verwenden wir ja docker, den wir immer wieder weg werfen.
dann haben wir jetzt nicht schon unnötige files rum liegen nur weil wir zu beginn annahmen getroffen haben, die wir dann wieder verworfen haben ;)

Reply:
Das regelt das Migrations-Tool von prisma.
Prisma führt eine Tabelle in der DB, in der die Migrations-Steps abgebildet werden, und dann auf die jeweilige DB angewendet.

Reply:
ja wie das funktioniert ist mir schon klar, ich arbeite ja auch mit Migrationen in Java.
ich meinte nur, man kann die ja eigentlich wieder zurücksetzen weil noch gibt es ja nicht wirklich was zu migrieren :D

Reply:
Das können wir natürlich machen. Ich persönlich denke dass es in diesem kleinen Projekt nicht viele Migrationen geben wird, darum habe ich dass mal belassen.